### ISSUE TITLE
Default to the same source image for all seasons/rotations

### ISSUE THREAD

**Issue by Leartin** [An-dz/tilecutter#9]
Currently, you have to define the image source for each rotation. I think it would be beneficial if the image source you set for South Summer Backimage would be automatically set for everything else.
If the user is like me, and produces rotation or even seasons in the same file, it saves some clicking.
If the user has all states of the buildings in different files, they need to manually set them just like before.

A more advanced version would be to let the user choose a default pattern. That is, once they choose the size and position in the South Summer Backimage, Tilecutter would already know where the other images would be (eg. "one 'step' [(x+y)*paksize/2] to the right" or "append '_snowy' to image source name" etc.)

**Comment by An-dz** [comment]
You're right, we generally put everything on a single png. For starters I'll make the first image you select on a new project apply to all bitmaps.
